### DISCUSSION TITLE
Custom Map Support

### DISCUSSION THREAD

**Discussion by treyyoder** [treyyoder/quakejs-docker#23] [Ideas]
Moving the custom map issue to a discussion: https://github.com/treyyoder/quakejs-docker/issues/7

**Comment by picatz** [comment]
👋 I don't have any feedback to share on how to implement this right now, but I just wanted to add that I would certainly use it if it was implemented! Would also be happy to provide feedback or testing.

Just for context I use this regularly, at least 3 times a year, to run Quake sessions:
https://github.com/picatz/terraform-google-quakejs
